ssas/hash_test.go

```diff
@@ -65,6 +67,35 @@ func (s *HashTestSuite) TestHashWithIter() {
 	s.True(h1.IsHashOf(val))
 }
 
+func TestHashIterTime(t *testing.T) {
+	// This test is not intended to run with the remain suite.  Is is solely used to determine the number of hash
+	// iterations needed to reach a 1 second duration. To invoke the test use:
+	// DEBUG=true  go test -v github.com/CMSgov/bcda-ssas-app/ssas -run TestHashIterTime
+
+	// Check for local dev environment
+	if os.Getenv("DEBUG") != "true" {
+		t.SkipNow()
+	}
+
+	var resultTime int64
```

Comment:
Nit 2 -Instead of resultTime, what do you think about totalTime? Since we're accumulating the various resultTimes?

Reply:
Renamed to totalTime